pkg/server/server.go

```diff
@@ -127,26 +127,17 @@ func (s *Server) ListenAndServe(ctx context.Context, logger log.Logger, addr str
 	grpc_health.RegisterHealthServer(srv, s.grpcProbe.HealthServer())
 
 	internalMux := chi.NewRouter()
-	if pathPrefix != "" {
-		internalMux.Mount(pathPrefix+"/api", grpcWebMux)
-	}
-	internalMux.Mount("/api", grpcWebMux)
```

Comment:
We could use a rewrite rule, but what's the purpose of this then? It feels like the other paths were overlooked. Otherwise, I'd remove this if-block and we do not change anything in the router when `pathPrefix` is set. WDYT?